Dockerfile

```diff
@@ -1,22 +1,30 @@
 ARG BUILD_IMAGE=build_default
+ARG GOLANG_VERSION=1.16-buster
+
 #
 # Default build environment for standard Tendermint chains
 #
-FROM golang:1.16-buster AS build_project
+FROM golang:${GOLANG_VERSION} AS build_project
+
+ARG APT_INSTALL_EXTRA_DEPS
 
 RUN apt-get update && \
   apt-get install --no-install-recommends --assume-yes curl unzip && \
   apt-get clean
 
+RUN apt-get install --no-install-recommends --assume-yes ${APT_INSTALL_EXTRA_DEPS}
```

Comment:
Could we just move this into the above `apt-get install` (after `unzip`) - then it's perfect

Reply:
should be solved now